src/commands/environments/copy-layout.js

```diff
@@ -43,8 +43,12 @@ class CopyLayoutCommand extends Command {
     }
 
     if (!answers || answers.confirm === toEnvironment.name) {
-      await manager.copyLayout(fromEnvironment.id, toEnvironment.id);
-      console.log(`Environment's layout ${chalk.red(fromEnvironment.name)} successfully copied to ${chalk.red(toEnvironment.name)}.`);
+      const copyLayout = await manager.copyLayout(fromEnvironment.id, toEnvironment.id);
+      if (copyLayout) {
+        console.log(`Environment's layout ${chalk.red(fromEnvironment.name)} successfully copied to ${chalk.red(toEnvironment.name)}.`);
+      } else {
+        logger.error('💀  Oops, something went wrong.💀');
```

Comment:
```suggestion
        this.error('💀  Oops, something went wrong.💀', { exit: 1 });
```

Reply:
This is not really good ? We don't know the errors. We should add another PR maybe  to handle correctly the errors

Reply:
Or do the same as in the create